### PR TITLE
cppreference search action

### PR DESCRIPTION
**Pull request by partouf**
Because why not

### CODE REVIEW

**Comment by mattgodbolt** [comment]
Why not indeed! This is great!

File: static/panes/editor.js [hunk 1]
```diff
@@ -706,6 +706,29 @@ Editor.prototype.initEditorActions = function () {
...
+    });
+};
+
+Editor.prototype.searchOnCppreference = function (ed) {
+    var pos = ed.getPosition();
+    var word = ed.getModel().getWordAtPosition(pos);
+    if (!word || !word.word) return;
+
+    var url = "https://en.cppreference.com/mwiki/index.php?search=" + encodeURIComponent(word.word);
+    window.open(url, '_blank');
```
Comment (apmorton) [inline static/panes/editor.js:731]: We should add `noopener` here - I believe in js it just goes in the third parameter to window.open, but please double check me on that.

Comment (partouf) [reply static/panes/editor.js:731]: Good one, will do

Comment (partouf) [reply static/panes/editor.js:731]: Checked mozilla docs and its indeed the third and accepts 'noopener'

**Pull request merged**